metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -716,6 +722,47 @@ class MetalsLanguageServer(
     onChange(paths).asJava
   }
 
+  private def insertPackage(path: AbsolutePath): Unit = {
+    def createPackageName(path: AbsolutePath): Option[String] = {
+      if (path.toFile.length() == 0) {
+        val sourceDirectory = buildTargets.inverseSourceDirectory(path)
+        val relativeDirectory = sourceDirectory
+          .map(path.toRelative)
+          .flatMap(relativePath => Option(relativePath.toFile.getParentFile))
```

Comment:
let's use `Path` whenever possible instead of the old `File`.
```scala
val relativeDirectory = sourceDirectory
          .map(path.toRelative)
          .flatMap(relativePath => Option(relativePath.toNIO.getParent))
```